### ISSUE TITLE
Failed to load leagues

### ISSUE THREAD

**Issue by refeshdeluxe** [SnosMe/awakened-poe-trade#1018]
I cant use awakened poe trade.
The app show:
Failed to load leagues
Make sure the realm is not under maintenance. Also try clicking on the "Browser" button, you may need to complete a CAPTCHA there.
 Can you help me fix it.
 Tks you so much sir

**Comment by wankio** [comment]
yeah Error: Unexpected token '<', "<!DOCTYPE "... is not valid JSON

it's cloudflare thing,
